core-plugins/src/main/java/io/cdap/plugin/batch/action/EmailAction.java

```diff
@@ -193,6 +212,12 @@ public static class Config extends ConditionConfig {
     @Macro
     private Boolean includeWorkflowToken;
 
+    @Nullable
+    @Description("Optional property that can be used to pass specific set of javamail properties. The syntax " +
+            "is 'property name:property value'. If this property is not set, default values are chosen.")
```

Comment:
nit: Please wrap the `<property_name>` and `<property_value>` in brackets for clarity, i.e. `<property_name>:<value>`, thanks!